calico_kubernetes/logutils.py

```diff
@@ -31,18 +33,14 @@ def configure_logger(logger, log_level, root_logger=False, log_dir=LOG_DIR):
                                                      maxBytes=10000000,
```

Comment:
I've just spotted that this isn't safe, since we'll be writing to this log file from multiple processes. Maybe look at https://pypi.python.org/pypi/ConcurrentLogHandler/0.9.1?